appendix-basic65.tex

```diff
@@ -5479,13 +5482,13 @@ \subsection{NEXT}
                is to increment the variable by 1.
                The index variable must be a real variable.
 
-               The {\bf start} value is used to initialise the index.
+               {\bf start} value to initialise the index with.
 
-               The {\bf end} value is used at the end of an iteration,
+               {\bf end} used at the end of an iteration,
```

Comment:
{\bf end} **is** used at the end...

Reply:
i changed it but not sure if it's what you wanted

Reply:
Oh, at the time, I had just wanted the addition of the 'is', but I probably wasn't verbose enough :D

Pondering this through again, maybe I'm 50/50 on the 'is' now, so I'll leave it up to you on that aspect.

I liked your more recent choice wording with 'checked', and would prefer you keep the 'of an iteration' part of it too. E.g.:

`{\bf end} checked at the end of an iteration, ...`

Reply:
re-added it